src/content/1.7/project/maintainers-guide/summary-github-lifecycle.md

```diff
@@ -10,7 +10,7 @@ This page quickly describes how [Issues](https://guides.github.com/features/issu
 
 ## The teams
 
-On GitHub, there are mainly three teams working together on the project.
+On GitHub, there are mainly three teams (employees of PrestaShop's company) working together on the project.
```

Comment:
I'm not sure about this, where does the Maintainers team fit in?

Reply:
I edited this part see https://github.com/PrestaShop/docs/pull/807/commits/7875579b171ffd8d381c7a41bee1aa8565dd75ff